Resources/Prototypes/_EinsteinEngines/Language/Standard/spaceitalian.yml

```
@@ -0,0 +1,153 @@
+# SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+# SPDX-FileCopyrightText: 2025 ImWeax <59857479+ImWeax@users.noreply.github.com>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Spoken by people from the Kitchen. Mama mia.
+- type: language
+  id: SpaceItalian
+  isVisibleLanguage: true
+  speech:
+    color: "#b54a4a"
+    fontId: NotoSansSC
+  obfuscation:
```

This does not need to be NotoSansSC. Leaving the `fontId` field empty will default to NotoSans, the current system the game is using. Since the replacement syllables are not Chinese, the font does not need to support it.
```suggestion
  speech:
    color: "#b54a4a"
  obfuscation:
```